listenbrainz/db/model/stats/user/artist.py

```diff
@@ -0,0 +1,39 @@
+import pydantic
+
+from datetime import datetime
+from enum import Enum
+from typing import Optional, List
+
+
+class StatisticsRange(Enum):
+    week = 'week'
+    month = 'month'
+    year = 'year'
+    all_time = 'all_time'
+
+
+class UserArtistRecord(pydantic.BaseModel):
+    artist_msid: Optional[str]
+    artist_mbids: List[str] = []
+    listen_count: int
+    artist_name: str
+
+
+class UserArtistStatForRange(pydantic.BaseModel):
```

Comment:
terminology: `UserArtistStatRange`. the `for` makes it sound more like a verb (method) rather than a noun (model)